forms-shared/src/summary-email/renderSummaryEmail.tsx

```diff
@@ -8,12 +8,12 @@ import { render } from '@react-email/components'
 import { SummaryEmail } from './SummaryEmail'
 import { FormDefinition } from '../definitions/formDefinitionTypes'
 
-export type RenderSummaryEmailFileIdUrlMap = Record<string, string>
+export type RenderSummaryEmailFileIdInfoMap = Record<string, { url: string; fileName: string }>
 
 export type RenderSummaryEmailPayload = {
   formDefinition: FormDefinition
   formData: GenericObjectType
-  fileIdUrlMap: RenderSummaryEmailFileIdUrlMap
+  fileIdUrlMap: RenderSummaryEmailFileIdInfoMap
```

Comment:
regarding the naming change: please rename the key `fileIdUrlMap` as well.

Reply:
thanks - browsed around code after this, found a few more places where the 'fileIdUrlMap' was still used - it should be replaced everywhere where it became a map of id -> url,fileName. Kept it in one place in tests where it's still id->url map.